packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts

```diff
@@ -71,79 +40,18 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
       } catch (e) {
         throw new Error(`The ChainID ${args.chainId} is not supported`)
       }
+    } else {
```

Comment:
i'd probably throw here rather than opaquely defaulting to bitcoin

Reply:
this was the behavior of the previous adapter, but will modify now

Reply:
@0xdef1cafe - lets chat there is specifically a test for "should return chainAdapter with default Bitcoin chainId if called with no chainId" so am concerned there was a reason for this.

Reply:
per discussion with @0xdef1cafe , killing this behavior and the associated test